docs/contributing/opensauced-writers-guide/how-to-add-your-post.md

```diff
@@ -0,0 +1,98 @@
+---
+id: writers-guide
+title: "OpenSauced Writers Guide"
+sidebar_label: "OpenSauced Writers Guide"
+keywords: 
+- "writers guide" 
+- "open source writers guide" 
+- "open source writer" 
+---
+
+Welcome to our Writers Contributor Guide. This will walk you through the steps of how to add your blog post to our community resources using the docs repository.
+
+## Repeat Authors: Adding to `authors.yml`
+
+For regular blog post authors, we should declare those authors globally in the configuration file:
+
+**Location:** `/blog/authors.yml`
+
+**Example:**
+```yaml
+BekahHW:
+  name: Bekah Hawrot Weigel
+  title: Developer Experience Lead
+  url: https://twitter.com/BekahHW
+  image_url: https://github.com/BekahHW.png
+  email: Bekah@opensauced.pizza
+  twitter: BekahHW
+
+nickytonline:
+  name: Nick Taylor
+  title: AI Engingeer
+  url: https://www.nickyt.co/
+  image_url: https://github.com/nickytonline.png
+```
+
+## Adding Blog Posts
+
+To publish in the blog, create a Markdown file within the blog directory. Note that we have subdirectories for each year, so writers should add their post to the folder that corresponds with the year.
+
+**Example Path:** `/blog/2024/06-03-welcome-opensauced.md`
+
+
+:::info
+
+You'll notice that many of the files include the year. These were created before we started using the year subdirectories. Either way works, but we're moving towards the year subdirectories for organization and omitting the year in the file name.
+
+:::
+
+**Markdown Example:**
+```markdown
+docs/blog/2024/06-03-welcome-opensauced.md
+---
+title: Welcome to OpenSauced
+description: This is my first post on OpenSauced.
+slug: welcome-opensauced
```

Comment:
Some folks might not know what slug is. We might want to give more information about this.

Even better, we might want to give all information on how to fill in the metadata. I imagine, folks would ask the same questions, such as: "What should I fill in the slug?" "What tags should I use here?" "What tags are available?" "Should I put my name or username?" and so on.

Reply:
We actually can't use tags until the docusaurus update gets merged in, but I'll add it in anywhat.